src/solver/conv_hip_implicit_gemm_fwd_v4r4_xdlops.cpp

```diff
@@ -99,8 +99,6 @@ bool PerformanceImplicitGemmForwardV4R4Xdlops::SetNextValue()
         }
         if(!NextFlag<false, true>(GemmBThreadCopyMoreGemmKPack))
             break;
-        if(!NextFlag<false, false>(GemmAThreadCopyMoreGemmK))
```

Comment:
Please add comment in accordance to https://github.com/ROCmSoftwarePlatform/MIOpen/pull/347#discussion_r512380380